.github/workflows/tests.yml

```
@@ -9,10 +9,15 @@ jobs:
   tests:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
+      - name: Check out code
+        uses: actions/checkout@v3
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
         with:
           node-version: 16
-      - uses: c-hive/gha-npm-cache@v1
-      - run: npm ci
-      - run: npm test
+          cache: 'npm'
+
+      - name: Run tests
+        run: npm ci
+        run: npm test
```

Can multiple run keys be specified like this or should it be done with a multiline value?
```suggestion
        run: |
            npm ci
            npm test
```